modules/administration-guide/pages/configuring-the-administrative-user.adoc

```diff
@@ -0,0 +1,36 @@
+:_content-type: PROCEDURE
+:navtitle: Configuring the administrative user
+:keywords: administration-guide, admin
+:page-aliases:
+
+[id="configuring-the-administrative-user_{context}"]
+= Configuring the administrative user
+
+The user with administrative privileges has all {prod-short} system permissions.
```

Comment:
It's not clear if the admin user is pre-created at installation time. And it's not clear if the CHE_SYSTEM_ADMIN_NAME should match the name of an existing user: what happens if the user doesn't exist? And what happens I specify ADMIN_NAME but not PRIVILEGED_MODE in CheCluster resource (i.e. is PRIVILEGED_MODE set to true by default)?